dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -423,6 +434,33 @@ public final class ThreadPoolController {
      */
     private int queueDepth = 0;
 
+    /**
+     * How many multiples of the current poolIncrement will qualify as a deep queue
+     */
+    private static final int deepQueueMultiple;
```

Comment:
The new static fields should be above the static initializer block above with the other static fields.

Reply:
done